__tests__/sanity.test.js

```diff
@@ -17,3 +17,15 @@ it('generates the right CSS', () => {
       expect(result.css).toBe(expected)
     })
 })
+
+it('generates the right CSS with implicit screen utilities', () => {
+  const input = fs.readFileSync(path.resolve(`${__dirname}/fixtures/tailwind-input-with-explicit-screen-utilities.css`), 'utf8')
+
+  return postcss([tailwind()])
+    .process(input)
+    .then(result => {
+      const expected = fs.readFileSync(path.resolve(`${__dirname}/fixtures/tailwind-output.css`), 'utf8')
+
+      expect(result.css).toBe(expected)
```

Comment:
This test is not quite proving what you want; try adding some extra CSS to the end of the input fixture to make sure that the responsive utilities all show up before that extra CSS and I think you'll see what I mean!

You'll need to update the output fixture too, or maybe duplicate it and add the new CSS to the end so we have separate fixtures for both scenarios.

Reply:
mmhhh, the problem here is that duplicating it would mean changing anything in two places in the future. Or am I wrong?

I understood that the test does not properly test my feature. I'll hack around a bit, but duplicatiing is not a good option IMO ?

Reply:
holy... this will get a mess.
When I do not duplicate everything (which is definitly bad for maintaining the library) I can't test it properly.

The problem is that the lib does not know if i used ```@tailwind utilities```.
I changed the test to meet my expectation.

Do you have a hint how you'd tackle this?
Thanks you very much for your advice!!

Reply:
Okay, I'm wrong.
Everything is fine, i just had wrong expectations.